sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryConstants.cs

```diff
@@ -19,6 +19,28 @@ internal static class QueryConstants
         public const string Top = "TOP";
         public const string Count = "COUNT";
 
-        // other query keywords added in this same fashion as they are implemented
+        public const string And = "AND";
+
+        public const string IsDefined = "IS_DEFINED";
+        public const string IsNull = "IS_NULL";
+        public const string StartsWith = "STARTSWITH";
+        public const string EndsWith = "ENDSWITH";
+        public const string IsOfModel = "IS_OF_MODEL";
+        public const string IsBool = "IS_BOOL";
+        public const string IsNumber = "IS_NUMBER";
+        public const string IsString = "IS_STRING";
+        public const string IsPrimative = "IS_PRIMATIVE";
+        public const string IsObject = "IS_OBJECT";
+
+        // Maps comparison operators represented alphabetically to respective symbolic representations.
+        public static Dictionary<string, string> ComparisonOperators = new Dictionary<string, string>()
```

Comment:
I am not sure if this dictionary really belongs here in the constants section but I can see why it's here. 
I would make it a read-only dictionary though ... no one is supposed to be able to change this.

Reply:
Agreed on `IReadOnlyDictionary` type, and `readonly` keyword